tests/test_logic.py

```diff
@@ -185,3 +185,47 @@ class ClassF(api.Collector):
 
     assert logic.plugins_by_families(
         [ClassD, ClassE, ClassF], ["a", "b", "c"]) == [ClassD, ClassE]
+
+
+REGISTERED_PATHS = []
```

Comment:
You can replace this with the decorator `@with_setup(lib.setup_empty, lib.teardown)`, which will empty any trace of a plug-in before calling the test, and restore things on exit. Much like what you've done here. See how it's used elsewhere to get an idea of how to use it.